communities/microgalaxy/lab/sections/2_microbial_isolates.yml

```diff
@@ -1,9 +1,409 @@
+      inputs:
+        - datatypes:
+            - phyloseq R object
+      button_link: "{{ galaxy_base_url }}/root?tool_id=interactive_tool_phyloseq"
+    - title_md: <code>RStudio</code> - An integrated development environment (IDE) for R and Python
+      description_md: >
+          This familiar R analysis software suite will let you explore your datasets in depth. Comes with ggplot2, RODBC, maps, shinyapps, knitr, LaTeX, bioconductor, cummeRbund, and many more pre-installed packages.
+      button_link: "{{ galaxy_base_url }}/root?tool_id=interactive_tool_rstudio"
+    - title_md: <code>Interactive JupyterLab Notebook</code> - A web-based interactive computing platform
+      description_md: >
+          The Jupyter Notebook is an open-source web application that allows you to create and share documents that contain live code, equations, visualizations and narrative text. Uses include: data cleaning and transformation, numerical simulation, statistical modeling, data visualization, machine learning, and much more.
+      inputs:
+        - datatypes:
+            - Jupyter Notebook
+      button_link: "{{ galaxy_base_url }}/root?tool_id=interactive_tool_jupyter_notebook"
+
+- id: databases
+  title: Databases
+  heading_md: Examples of databases prebuilt in some tools by microGalaxy. Even more databases are also found in microGalaxy, such as PhageDB, MAdLandDB, SwissProt, PDB, etc.
+  content:
+    subsections:
+      - id: genomic-databases
+        title: Genomic Databases
+        content:
+          - title_md: <code>NCBI RefSeq</code> - Reference genome database
+            button_link: "https://www.ncbi.nlm.nih.gov/refseq/"
+            button_tip: "Visit NCBI RefSeq"
+            description_md: >
+              A comprehensive collection of reference genomes for bacteria, archaea, and eukaryotes.
+              Used in tools such as [<code>Prokka</code>]({{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/crs4/prokka/prokka)
+          - title_md: <code>ENA</code> - European Nucleotide Archive
+            button_link: "https://www.ebi.ac.uk/ena"
+            button_tip: "Visit ENA"
+            description_md: >
+              A primary repository for nucleotide sequencing data.
+              Used in tools such as [<code>EBI Search</code>]({{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/ebi_search_rest_results/ebi_search_rest_results)
+
+      - id: transcriptomic-databases
+        title: Transcriptomic Databases
+        content:
+          - title_md: <code>GEO</code> - Gene Expression Omnibus
+            button_link: "https://www.ncbi.nlm.nih.gov/geo/"
+            button_tip: "Visit GEO"
+            description_md: >
+              A public functional genomics data repository for transcriptomic datasets.
+              Used in tools such as [<code>DESeq2</code>]({{ galaxy_base_url }}/root?tool_id=toolshed.g2.bx.psu.edu/repos/iuc/deseq2/deseq2)
```

Comment:
Is it really used in DESEq2?

Reply:
No you are right I will update the text same for the one below, they are both used after using datasets from the databases just to process it afterwards